layers/+readers/deft/packages.el

```diff
@@ -21,7 +61,11 @@
             deft-text-mode 'org-mode
             deft-use-filename-as-title t
             deft-use-filter-string-for-filename t)
-      (spacemacs/set-leader-keys "an" 'spacemacs/deft)
+      ;; in applications prefix, NOTE: backward incompatible keybindings
```

Comment:
Please change the keybinding for accessing normal `deft` to `ann` this is more idiomatic. This is repeating the category letter calls the most generic command of the category which would be vanilla `deft` mode.

Please also change the keybinding only if zettelmode is really loaded/used.